### PR TITLE
acceptance tests updated , run with beaker 4

### PR DESCRIPTION
**Pull request by Dan33l**
The goal of the PR is to have acceptance tests up to date with updated master branch.

### CODE REVIEW

**Comment by Dan33l** [comment]
WIP status removed on the PR.
Puppetlabs does not provide ubuntu 18.04 vbox. With the official ubuntu bionic vbox, i encountered the issue described here https://github.com/hashicorp/vagrant/issues/9524

File: Gemfile [hunk 1]
```diff
@@ -40,17 +40,18 @@ group :development do
...
 group :system_tests do
-  gem 'winrm',                              :require => false
-  if beaker_version = ENV['BEAKER_VERSION']
-    gem 'beaker', *location_for(beaker_version)
-  else
-    gem 'beaker', '>= 3.9.0', :require => false
-  end
-  if beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
-    gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  gem 'pry'
```
Comment (bastelfreak) [inline Gemfile]: all those changes will be overwritten. I assume you used pry only once and it isn't needed always. Can you remove it from the PR? All Gemfiles are generated from this file: https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml. You can provide a PR to that repo if you like. Maybe it is a good idea to add the pry gems to the [development](https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml#L62) group.

File: Gemfile [hunk 2]
```diff
@@ -40,17 +40,18 @@ group :development do
...
+  if RUBY_VERSION =~ /^2/
+    gem 'pry-byebug'
   else
-    gem 'beaker-rspec',  :require => false
+    gem 'pry-debugger'
   end
+  gem 'winrm',                              :require => false
+  gem 'beaker',                             :require => false
+  gem 'beaker-puppet',                      :require => false
+  gem 'beaker-vagrant',                     :require => false
```
Comment (bastelfreak) [inline Gemfile]: `beaker-vagrant` needs to be added to the `.sync.yml` file. An example can be found here:
https://github.com/voxpupuli/puppet-grafana/blob/79745359d28fc476fc2aab30970df442a43e6463/.sync.yml#L7-L10

Comment (Dan33l) [reply Gemfile]: done

File: Gemfile (hunk 1 above)
Comment (Dan33l) [reply Gemfile]: all pry stuffs are removed.

File: README.md [hunk 3]
```diff
@@ -15,18 +15,15 @@ By default, this module will install the Oracle VirtualBox yum/apt repo, install
...
-- Debian 6.0.10
-- Debian 7.8
-- Ubuntu 10.04
-- Ubuntu 12.04
-- Ubuntu 14.04
+- CentOS 6
+- CentOS 7
+- Debian 8
+- Debian 9
+- Ubuntu 16.04
```
Comment (dhollinger) [inline README.md]: Should we add support for Ubuntu 18.04?

**Pull request merged**